client/reader/following/intro.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+
+const FollowingIntro = ( { translate } ) => {
+	const isNewReader = true;
+
+	if ( ! isNewReader ) {
+		return null;
+	}
+
+	return (
+		<header className="following__intro">
+			<h1 className="following__intro-title">{ translate( 'This is Reader' ) }</h1>
+			<p className="following__intro-description">
+				{ translate( 'Reader is a customizable magazine of stories from WordPress.com and across the web. Follow a site and their latest posts will appear here.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Reader is a customizable magazine of stories from WordPress.com and across the web. Follow a few sites and their latest posts will appear here. To begin, tell us something you like.' )` *ES Score: 13.13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).